DCOS/regression-testing/src/vendor/github.com/Azure/acs-engine/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Is the MIT license compatible with the Apache2 license from this repository ? We'll be having dual license if we merge like this.

Reply:
Looks like there are no restrictions:
https://www.quora.com/Is-the-MIT-license-compatible-with-the-Apache-License-Version-2-APLv2